website/blog/2024-07-25-ci-migration.mdx

```diff
@@ -0,0 +1,78 @@
+---
+slug: ci-migration
+title: "Optimizing and Migrating Velox CI Workloads to Github Actions"
+authors: [jwujciak, kgpai]
+tags: [tech-blog, packaging]
+---
+
+<figure>
+ <img src="/img/abstract-white-background.jpg" height= "100%" width="100%"/>
+</figure>
```

Comment:
While a nice image, I am not sure it really adds anything?

Reply:
yep, that's correct. We can remove it as well. Please let me know.

Reply:
cc @kgpai for thoughts